torchvision/csrc/deform_conv2d.cpp

```diff
@@ -77,6 +79,10 @@ at::Tensor deform_conv2d_autocast(
              use_mask)
       .to(input.scalar_type());
 }
+
+TORCH_LIBRARY_IMPL(torchvision, Autocast, m) {
+  m.impl("deform_conv2d", deform_conv2d_autocast);
```

Comment:
nit: I didn't know that the `TORCH_LIBRARY_IMPL` can actually be present before `TORCH_LIBRARY_FRAGMENT`. This is just a comment, no need to change the code.

Also, I wonder if for the future we should create new folders for `autograd` and `autocast`, like we have for `cpu` and `cuda`?

Reply:
Yes indeed. The order looks a bit strange. 

The reason I put it like that was to minimize the number of `#if` blocks AND avoid moving large blocks of code around and lose the git history. If we don't mind about the latter, we can easily refactor it. :)

Reply:
Let's leave it like this for now. If we decide to create `autocast` and `autograd` folders, then things will be nicely present there (and no ifdefs in the code)